content/docs/started/kubeflow-overview.md

```diff
@@ -0,0 +1,151 @@
+producing the results you need.
+
+For the sake of simplicity, the following diagram
+shows the workflow stages in sequence, with just one arrow at the end pointing
+back into the flow as an indication of the iterative nature of the process:
+
+<img src="/docs/images/kubeflow-overview-workflow-diagram-1.svg" 
+  alt="A typical machine learning workflow"
+  class="mt-3 mb-3 border border-info rounded">
+
+Looking at the stages in more detail:
+
+* In the experimental phase, you develop your model based on initial
+  assumptions, and test and update the model iteratively to produce the
+  results you're looking for:
+
+  * Identify the problem you want the ML system to solve.
+  * Collect and analyse the data you need to train your ML model.
+  * Choose an ML framework and algorithm, and code the initial version of your 
+    model.
+  * Experiment with the data and with training your model.
+  * Tune the model hyperparameters to ensure the most efficient processing and the
+    most accurate results possible.
+
+* In the production phase, you deploy a system that performs the following 
+  processes:
+
+  * Transform the data into the format that your training system needs. The
+    transformation process must be the same in the experimental and production
+    phases, to ensure that the ML model bases its predictions on the same data 
+    format as that used during training.
+  * Train the ML model.
+  * Serve the model for online prediction or for running in batch mode.
+  * Monitor the model's performance, and feed the results into your processes
+    for tuning or retraining the model.
+
+## Kubeflow components in the ML workflow
+
+The next diagram adds Kubeflow to the workflow, showing which Kubeflow
+components are useful at each stage:
+
+<img src="/docs/images/kubeflow-overview-workflow-diagram-2.svg" 
+  alt="Where Kubeflow fits into a typical machine learning workflow"
+  class="mt-3 mb-3 border border-info rounded">
+
+To learn more, read the following guides to the Kubeflow components:
+
+* Kubeflow includes services for spawning and managing 
+  [Jupyter notebooks](/docs/notebooks/). Use notebooks for interactive data 
+  science and experimenting with ML workflows.
+
+* [Kubeflow Pipelines](/docs/pipelines/pipelines-overview/) is a platform for 
+  building, deploying, and managing multi-step ML workflows based on Docker 
+  containers.
+
+* Kubeflow offers a number of [components](/docs/components/) that you can use
+  to build your ML training, hyperparameter tuning, and serving workloads across
+  multiple platforms.
+
+## Example of a specific ML workflow
+
+The following diagram shows a simple example of a specific ML workflow that you
+can use to train and serve a model trained on the MNIST dataset:
+
+<img src="/docs/images/kubeflow-gcp-e2e-tutorial.svg" 
+  alt="ML workflow for training and serving an MNIST model"
+  class="mt-3 mb-3 border border-info rounded">
```

Comment:
I suggest we could simplify the diagram on this section to map to the above diagram - 'kubeflow-overview-workflow-diagram-2'.

Reply:
Thanks @krishnadurai I did try to map this diagram to the one you mention. Do you have ideas on how to do it better? The source of the diagrams is in this folder in the Kubeflow public drive:
https://drive.google.com/corp/drive/folders/1BMnjzP3nn1IRopdlts1LTHNQq9fU97s5

If you like, you can copy the slide and suggest some simplifications that way.

Reply:
@sarahmaddox I took a stab at it.
I tried to link this diagram to the one which we have above -  'kubeflow-overview-workflow-diagram-2'. Though I would require your help to complete this:
https://docs.google.com/presentation/d/1TFh2a41ALGxMhXgI9ALbcKoHPmrs-O7SkIA9oBJCA3g/edit?usp=sharing

I'm not sure which components we have in Kubeflow which map to these exact steps: Transform Data and Monitor a model's performance.

You could take a call if you like this better or we could stick to the earlier version.

Reply:
Thanks @krishnadurai that's super useful. I've taken your diagram as a basis, and adjusted it slightly. You can see the result here:
https://deploy-preview-1339--competent-brattain-de2d6d.netlify.com/docs/started/kubeflow-overview/#example-of-a-specific-ml-workflow

The file is also in the same Kubeflow public drive.

Reply:
I think this is great. I reached out to the KFServing team and they suggested that metrics using [graphana](https://github.com/knative/docs/blob/master/docs/serving/accessing-metrics.md) is a way to monitor models rather than Stackdriver's infrastructure metrics.

Reply:
Thanks @krishnadurai that's interesting. I'll stick to Stackdriver for now, since the specific sample linked in the doc doesn't mention Grafana. We should consider creating a sample that does use Grafana - perhaps in our upcoming doc sprint!